### DISCUSSION TITLE
CUSTOM ANIMATION

### DISCUSSION THREAD

**Discussion by cho66x** [naverz/zepeto-studio-global#44] [General]
For custom animation, is it possible to use the animation source from mixamo?

**Comment by naverz-mkao** [comment]
Hi Anggacho,

Yes, you can use the animation source from Mixamo!

**1:) Once you save your animation from Mixamo, just drag it into your Assets folder.**

**2:) Click on your Mixamo file (Mine is called "Capoeira"), and click the "Rig" tab in the inspector window.**

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/107931355/184661130-7e9d102f-e8a4-4358-86eb-54fd8ad1bd64.png">

**3:) Change your animation type to "Humanoid" and hit "Apply".**

<img width="513" alt="image" src="https://user-images.githubusercontent.com/107931355/184662556-58668cb7-b0a0-43dc-90ae-e460b4fd12d1.png">

**4:) You may need to change the "Motion" option to "Root Transform" depending on the animation.**

<img width="512" alt="image" src="https://user-images.githubusercontent.com/107931355/184664950-017aac20-56c6-47db-a89f-720ccaf8e4c7.png">

**5:) You should now be able to use the animation on your Zepeto player!**

**Comment by cho66x** [comment]
Thank you, I have solved the problem. your answer is very helpful

**Reply by naverz-mkao** [reply]
Glad to help Anggacho!
